<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_widget</class>
 <widget class="QWidget" name="main_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compare</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
        border-radius: 4px;
        background-color: rgb(88, 88, 88);
        color: rgb(255, 255, 255);
        }

        QMenu::item:selected {
        background-color: rgb(23, 111, 204);
        }

        QMenu::item:disabled {
        color: rgba(255, 255, 255, 75);
        }</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stacked_widget">
     <widget class="QWidget" name="page_input">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QWidget" name="text_entry_area" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="meta_box" native="true">
            <property name="styleSheet">
             <string notr="true">/*QWidget{
border: 1px solid black;
border-color: rgb(77, 77, 77);
}

QLabel{
border:none;
}*/</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="quest_label">
               <property name="text">
                <string>Meta Quest 2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="meta_text_box">
               <property name="styleSheet">
                <string notr="true">QPlainTextEdit {
                      background-color: rgb(42, 42, 42);
                      padding: 5px;
                      spacing: 10px;
                      border: 1px solid;
                      border-color: rgba(255, 255, 255, 30);
                      border-right-color: rgba(0, 0, 0, 128);
                      border-bottom-color: rgba(0, 0, 0, 128);
                      color: rgb(255, 255, 255);
                      border-radius: 4px;
                      border: 1px solid;
                      border-color: rgb(83, 83, 83);
                      }</string>
               </property>
               <property name="plainText">
                <string/>
               </property>
               <property name="backgroundVisible">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>Enter the privacy policy text here</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="apple_box" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="quest_label_2">
               <property name="text">
                <string>Apple Vision Pro</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="apple_text_box">
               <property name="styleSheet">
                <string notr="true">QPlainTextEdit {
                      background-color: rgb(42, 42, 42);
                      padding: 5px;
                      spacing: 10px;
                      border: 1px solid;
                      border-color: rgba(255, 255, 255, 30);
                      border-right-color: rgba(0, 0, 0, 128);
                      border-bottom-color: rgba(0, 0, 0, 128);
                      color: rgb(255, 255, 255);
                      border-radius: 4px;
                      border: 1px solid;
                      border-color: rgb(83, 83, 83);
                      }</string>
               </property>
               <property name="plainText">
                <string/>
               </property>
               <property name="backgroundVisible">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>Enter the privacy policy text here</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="sony_box" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="quest_label_5">
               <property name="text">
                <string>Sony PS VR 2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="sony_text_box">
               <property name="styleSheet">
                <string notr="true">QPlainTextEdit {
                      background-color: rgb(42, 42, 42);
                      padding: 5px;
                      spacing: 10px;
                      border: 1px solid;
                      border-color: rgba(255, 255, 255, 30);
                      border-right-color: rgba(0, 0, 0, 128);
                      border-bottom-color: rgba(0, 0, 0, 128);
                      color: rgb(255, 255, 255);
                      border-radius: 4px;
                      border: 1px solid;
                      border-color: rgb(83, 83, 83);
                      }</string>
               </property>
               <property name="plainText">
                <string/>
               </property>
               <property name="backgroundVisible">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>Enter the privacy policy text here</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="summarize_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                      background-color: qlineargradient(spread:repeat, x1:1, y1:1, x2:1, y2:0,
                      stop:0 rgba(60, 60, 60, 255), stop: 1 rgba(80, 80, 80, 255));
                      padding: 5px;
                      spacing: 10px;
                      border: 1px solid;
                      border-color: rgba(255, 255, 255, 30);
                      border-right-color: rgba(0, 0, 0, 128);
                      border-bottom-color: rgba(0, 0, 0, 128);
                      color: rgb(255, 255, 255);
                      }
                      QPushButton:hover {
                      background-color: qlineargradient(spread:repeat, x1:1, y1:1, x2:1, y2:0,
                      stop:0 rgba(80, 80, 80, 255), stop: 1 rgba(100, 100, 100, 255));
                      border: 1px solid;
                      border-color: rgba(255, 255, 255, 80);
                      border-right-color: rgba(0, 0, 0, 255);
                      border-bottom-color: rgba(0, 0, 0, 255);
                      }
                      QPushButton:pressed {
                      background-color: qlineargradient(spread:repeat, x1:1, y1:1, x2:1, y2:0,
                      stop:1 rgba(60, 60, 60, 255), stop: 0 rgba(70, 70, 70, 255));
                      border-color: rgba(0, 0, 0, 150);
                      border-right-color: rgba(255, 255, 255, 50);
                      border-bottom-color: rgba(255, 255, 255, 50);
                      }
                      QPushButton::indicator:unchecked {
                      background-color: rgb(38, 38, 38);
                      border-radius: 4px;
                      }

                      QPushButton::indicator:checked {
                      border-radius: 4px;
                      background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5,
                      fx:0.5, fy:0.5, stop:0.357513 rgba(200, 200, 200, 255), stop:0.388601 rgba(38,
                      38, 38, 255));
                      }</string>
         </property>
         <property name="text">
          <string>Compare</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_output">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="result_label">
            <property name="styleSheet">
             <string notr="true">QLabel{
                            border-radius: 4px;
                            background-color: qlineargradient(spread:repeat, x1:1, y1:1, x2:1, y2:0,
                            stop:1 rgba(40, 40, 40, 255), stop:0 rgba(50, 50, 50, 255));
                            border: 1px solid;
                            border-color: rgb(83, 83, 83);
                            color: rgb(255, 255, 255);
                            padding: 5px;
                            }</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                      background-color: qlineargradient(spread:repeat, x1:1, y1:1, x2:1, y2:0,
                      stop:0 rgba(60, 60, 60, 255), stop: 1 rgba(80, 80, 80, 255));
                      padding: 5px;
                      spacing: 10px;
                      border: 1px solid;
                      border-color: rgba(255, 255, 255, 30);
                      border-right-color: rgba(0, 0, 0, 128);
                      border-bottom-color: rgba(0, 0, 0, 128);
                      color: rgb(255, 255, 255);
                      }
                      QPushButton:hover {
                      background-color: qlineargradient(spread:repeat, x1:1, y1:1, x2:1, y2:0,
                      stop:0 rgba(80, 80, 80, 255), stop: 1 rgba(100, 100, 100, 255));
                      border: 1px solid;
                      border-color: rgba(255, 255, 255, 80);
                      border-right-color: rgba(0, 0, 0, 255);
                      border-bottom-color: rgba(0, 0, 0, 255);
                      }
                      QPushButton:pressed {
                      background-color: qlineargradient(spread:repeat, x1:1, y1:1, x2:1, y2:0,
                      stop:1 rgba(60, 60, 60, 255), stop: 0 rgba(70, 70, 70, 255));
                      border-color: rgba(0, 0, 0, 150);
                      border-right-color: rgba(255, 255, 255, 50);
                      border-bottom-color: rgba(255, 255, 255, 50);
                      }
                      QPushButton::indicator:unchecked {
                      background-color: rgb(38, 38, 38);
                      border-radius: 4px;
                      }

                      QPushButton::indicator:checked {
                      border-radius: 4px;
                      background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5,
                      fx:0.5, fy:0.5, stop:0.357513 rgba(200, 200, 200, 255), stop:0.388601 rgba(38,
                      38, 38, 255));
                      }</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
